sources/vega/commands/v1/commands.proto

```diff
@@ -128,10 +128,15 @@ message DelegateSubmission {
   uint64 amount = 2;
 }
 
-// A command to submit an instruction to undelete stake from a node
-message UndelegateAtEpochEndSubmission {
-  // The ID for the node to undelegate from
+message UndelegateSubmission {
+  enum Method {
+    Now = 0;
+    AtEndOfEpoch =1;
+    InAnger = 2;
+    // more to come eventually
```

Comment:
I realise I wrote the sample fo this initially, but the protobuf linting we are using will require these enum variant to be all named in SNAKE_CASE_CAPITALISED, prefixed with the name of enum METHOD_*,  start part a variant name METHOD_UNSPECIFIED

Reply:
you're right I missed that